docs/Installation-Guide.rst

```diff
@@ -670,6 +670,23 @@ On Linux Java wrapper of LightGBM can be built using **Java**, **SWIG**, **CMake
      cmake -DUSE_SWIG=ON ..
      make -j4
 
+macOS
+^^^^^
+
+On macOS Java wrapper of LightGBM can be build using **Java**, **SWIG**, **CMake** and **gcc** or **Apple Clang**
```

Comment:
@imatiach-msft Wow! Great news!

`built`?

Have you really tested the both cases: with `gcc` and with `Apple Clang`?

Reply:
will update comment - haven't tested with Clang, only gcc, probably should try to test it

Reply:
OK. If everything is OK, please follow macOS style guide: separate sections for Apple Clang and gcc, and also special case for Apple Clang on Mojave due to OpenMP issue: https://github.com/Microsoft/LightGBM/blob/master/docs/Installation-Guide.rst#macos

Reply:
thanks, I've updated the installation instructions.  I still have to test on Apple Clang, but I don't have a mac at home so it may be a couple days before I will get a chance to do so at work on one of the shared test machines (unless someone with a mac could test the changes in this PR).